### PR TITLE
UMIP #113: Whitelist BPRO as collateral

### PR DESCRIPTION
**Pull request by yaronvel**

### CODE REVIEW

File: UMIPs/umip-113.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+**UMIP #113**
```
Comment (Reinis-FRP) [inline UMIPs/umip-113.md:1]: Leave the UMIP # empty now as it would be assigned when merging the PR

File: UMIPs/umip-113.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+-   **UMIP title:** Add **BPRO** as collateral currency 
+-   **Author**  yaron@bprotocol.org
+-   **Status: Draft**
+-   **Created:**  7.7.2021
+-   **Discourse Link:**  Insert link to discourse topic after it has been moved into draft UMIPs
```
Comment (Reinis-FRP) [inline UMIPs/umip-113.md:7]: Please add the link

File: UMIPs/umip-113.md (hunk 1 above)
Comment (yaronvel) [reply UMIPs/umip-113.md]: done

File: UMIPs/umip-113.md (hunk 2 above)
Comment (yaronvel) [reply UMIPs/umip-113.md]: done

File: UMIPs/umip-113.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Implementation
+
+
+This change has no implementations other than the aforementioned governor transactions
+
+## Security considerations
+
+Adding a collateral currency introduces a level of risk into the UMA Ecosystem.  This collateral type should be monitored to ensure that the proposed collateral continues to have value.
+
+Contract deployers considering using this collateral in an UMA contract should refer to the guidelines on collateral type usage available here (insert docs link) to ensure appropriate use.
```
Comment (Reinis-FRP) [inline UMIPs/umip-113.md:38]: Collateral type usage guidelines link was missing in the original template (will update there shortly), but please update it here
```suggestion
Contract deployers considering using this collateral in an UMA contract should refer to the [guidelines on collateral type usage](https://docs.umaproject.org/uma-tokenholders/guidence-on-collateral-currency-addition) to ensure appropriate use.
```

Comment (yaronvel) [reply UMIPs/umip-113.md]: done

File: UMIPs/umip-113.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+## Motivation
+
+The addition of this collateral currency offers additional functionality to the UMA protocol and increases the range of contracts that can be built.
+
+## Technical Specification
+
+To accomplish this upgrade, the following changes need to be made:
+
+-   The **BPRO** address **[0xbbbbbbb5aa847a2003fbc6b5c16df0bd1e725f61](https://etherscan.io/token/0xbbbbbbb5aa847a2003fbc6b5c16df0bd1e725f61)** needs to be added to the collateral currency whitelist introduced in UMIP-8.
+-   A final fee of **150 BPRO** needs to be added for **BPRO** in the Store contract.
```
Comment (smb2796) [inline UMIPs/umip-113.md:22]: This is a little low. Can you set this to 200 BPRO?

Comment (yaronvel) [reply UMIPs/umip-113.md]: done

**Review by smb2796 — APPROVED** [review]

**Pull request merged**